code/modules/mob/living/silicon/pai/software.dm

```diff
@@ -253,22 +256,28 @@
 			if("securityhud")
 				if(href_list["toggle"])
 					secHUD = !secHUD
```

Comment:
check if they actually have program downloaded for all the huds so they can't href exploit it